src/main/java/org/broadinstitute/hellbender/tools/spark/sv/ReadMetadata.java

```diff
@@ -28,9 +28,9 @@
     private final Map<String, Integer> contigNameToID;
     private final long nReads;
     private final long maxReadsInPartition;
-    private final int nPartitions;
     private final int coverage;
-    private final Map<String, ReadGroupFragmentStatistics> readGroupToFragmentStatistics;
+    private final PartitionBounds[] partitionBounds;
+    private final Map<String, LibraryFragmentStatistics> readGroupToFragmentStatistics;
```

Comment:
The fragment length stats are still looked up by read group.  It's just that they're accumulated by library, and then the same set of statistics is shared over all the groups derived from that library.
I'll say that in the doc.